lib/fastly/client.rb

```diff
@@ -84,22 +86,36 @@ def put(path, params = {})
     end
 
     def delete(path)
-      resp  = http.delete(path, headers)
+      extras = params.delete(:headers) || {}
```

Comment:
Same comment as with `get`. Also, will need to be changed to `def delete(path, params = {})` or it will blow up